cognite/client/_api/datapoint_tasks.py

```diff
@@ -31,12 +30,24 @@
     cast,
 )
 
-from google.protobuf.message import Message
+from google.protobuf.internal.containers import RepeatedCompositeFieldContainer
 from typing_extensions import NotRequired, TypeAlias
 
 from cognite.client._proto.data_point_list_response_pb2 import DataPointListItem
-from cognite.client._proto.data_points_pb2 import AggregateDatapoint, NumericDatapoint, StringDatapoint
-from cognite.client.data_classes.datapoints import NUMPY_IS_AVAILABLE, Aggregate, Datapoints, DatapointsArray
+from cognite.client._proto.data_points_pb2 import (
```

Comment:
This module is difficult to follow. It seems to be some arguments passed in multiple levels. Is this something that can be refactored in the future? Or it is just complexity that is necessary due to the complexity of what the SDK is doing on top of the datapoint endpoint?

Reply:
With DatapointsQuery, I'm planning to clean up a lot... but even with that, there will be quite a bit of nasty logic remaining due to how many fetch patterns we have optimized routines for.